sgx-sys/src/lib.rs

```diff
@@ -58,6 +58,8 @@ mod test {
         secs.base = 8192;
         secs.ssa_frame_size = 4096;
         secs.attributes = secs::Attributes::MODE_64_BIT;
+        secs.miscselect = secs::MiscSelect::EXINFO;
+        secs.xfrm = 0x3;
```

Comment:
First off, awesome job!

The big question is: what belongs in `xfrm`?

Currently, `xfrm` is defined as a `u64` (https://github.com/enarx/enarx/blob/master/sgx-sys/src/secs.rs#L50). This probably isn't correct. Is this a bundle of flags? If so, we should turn this into a bitflags type (see: https://github.com/enarx/enarx/blob/master/sgx-sys/src/secs.rs#L28).

Reply:
Thanks - I'm looking into this today.

Reply:
@lkatalin I can confirm that your patch indeed does fix the tests.

Reply:
@npmccallum That's great! 

I turned the `xfrm` into a set of bigflags based on the description of `xcr0` from https://en.wikipedia.org/wiki/Control_register and turned on the last two bits, so the test still passes. I'm still not sure if this is the correct value. I may send an email about it.

Reply:
Good enough for me.